config/Configured.sol

```diff
@@ -65,6 +65,22 @@ contract Configured is StdChains {
         );
     }
 
+    /// @dev Slicing bytes array data is only possible with bytes in calldata, but internal functions cannot create calldata.
+    ///      So we work around this by doing an external call on this contract to force the data to be in calldata.
+    function _unwrapTxData(bytes memory data) internal view returns (Transaction memory) {
+        return this.unwrapTxData(data);
+    }
+
+    function unwrapTxData(bytes calldata data) external pure returns (Transaction memory transaction) {
+        (transaction.to, transaction.value, transaction.data, transaction.operation) = abi.decode(data[4:], (address, uint256, bytes, Operation));
+    }
```

Comment:
I wanted to leave it there because it is basically only to be used internally. Because this is not a deployed contract and is basically just a testing hack, I think it's easier to reason about when they are grouped together.

Reply:
as you want then